reference/providers/oracle.md

```diff
@@ -1,8 +1,90 @@
+
+## Resource mapping
+
+### Account
+The Oracle Cloud provider in Spinnaker maps a Spinnaker [__Account__](/concepts/providers/#accounts)
+to an [Oracle Cloud Infrastructure user]( https://cloud.oracle.com/en_US/tryit) in 
+[Oracle Cloud Infrastructure](https://cloud.oracle.com/) (OCI). 
+Spinnaker authenticates itself with Oracle Cloud using OCI user credentials - 
+see the [setup guide](/setup/providers/oracle). 
+
+### Load Balancer
+A Spinnaker **load balancer** maps to an [OCI Load Balancer](https://docs.cloud.oracle.com/iaas/Content/Balance/Concepts/balanceoverview.htm).
+
+The OCI Load Balancing service offers a load balancer with your choice of a public or private IP
+address, and provisioned bandwidth. The essential components for load balancing include:
+* A load balancer with pre-provisioned bandwidth.
+* A backend set with a health check policy. See [Managing Backend Sets](https://docs.cloud.oracle.com/iaas/Content/Balance/Tasks/managingbackendsets.htm).
+* One or more listeners . See [Managing Load Balancer Listeners](https://docs.cloud.oracle.com/iaas/Content/Balance/Tasks/managinglisteners.htm).
+* Optionally, you can associate your listeners with SSL server certificates to manage how your system handles SSL traffic. See [Managing SSL Certificates](https://docs.cloud.oracle.com/iaas/Content/Balance/Tasks/managingcertificates.htm).
+
+### Server Group
+A Spinnaker **server group** maps to a group of [OCI Virtual Machine Instances](https://docs.cloud.oracle.com/iaas/Content/Compute/Concepts/computeoverview.htm).
+
+### Instance
+A Spinnaker **instance** maps to an [OCI Virtual Machine Instance](https://docs.cloud.oracle.com/iaas/Content/Compute/Concepts/computeoverview.htm).
+
+When you launch an instance, you choose the most appropriate shape which is a template describing 
+the number of CPUs, amount of memory, and other resources allocated to a newly created instance. 
+See [Compute Shapes](https://docs.cloud.oracle.com/iaas/Content/Compute/References/computeshapes.htm) 
+for a list of available VM shapes.
+
+### Firewall
+In OCI, a [security list](https://docs.cloud.oracle.com/iaas/Content/Network/Concepts/securitylists.htm) 
+provides a virtual firewall for an instance, with ingress and egress rules that specify the types 
+of traffic allowed in and out. Each security list is enforced at the instance level.
+
+The Oracle Cloud provider in Spinnaker currently does not support the ability to create firewalls 
+for Oracle Cloud from Spinnaker. You can use the Oracle Cloud console to create and manage Security 
+Lists to provide a virtual firewall for an instance.
+
+## Operation mapping
+
+### Deploy
+
+Deploys a group of VM instances in OCI.
+
+A new server group template is created for each new server group and stored in Oracle Object 
+Storage. The Oracle Cloud provider in Spinnaker creates a bucket named `_spinnaker_server_group_data`
```

Comment:
Spinnaker clouddriver (Oracle provider) creates the "_spinnaker_server_group_data" bucket in Oracle Object Store within the user's tenancy.

Reply:
I would document that

Reply:
Yes, we do have that, "The Oracle Cloud provider in Spinnaker creates a bucket named _spinnaker_server_group_data to store all the server group templates." Or do I misunderstand what you suggest to add to this section?

Reply:
I change this to "The Oracle Cloud provider in Spinnaker creates a bucket named _spinnaker_server_group_data within the user's tenancy to keep track of all the server groups created."  This is also to avoid using the unexplained term of "server group template"

Reply:
Ah sorry I must've misread this or the context. LGTM